pkg/services/ngalert/api/api_alertmanager.go

```diff
@@ -438,6 +447,27 @@ func statusForTestReceivers(v []notifier.TestReceiverResult) int {
 	}
 }
 
+func newTestTemplateResult(res *notifier.TestTemplatesResults) apimodels.TestTemplatesResults {
+	apiRes := apimodels.TestTemplatesResults{
```

Comment:
Just do this instead of explicit `nil`?

```suggestion
	apiRes := apimodels.TestTemplatesResults{}
```